crs_support/urdf/swri_demo.urdf.xacro

```diff
@@ -235,145 +329,127 @@
         </inertial>
 
     </link>
-    <link name="base" />
-    <joint name="base_link-base_fixed_joint" type="fixed">
-        <origin rpy="-1.571 0 0" xyz="0 0 0" />
-        <parent link="base" />
-        <child link="base_link" />
+    <link name="${prefix}base" />
+
+    <joint name="table_to_ur" type="fixed">
+      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 3.1415"/>
+      <parent link="world"/>
+      <child link="${prefix}base"/>
     </joint>
-    <link name="tool0" />
-    <joint name="wrist_3_link-tool0_fixed_joint" type="fixed">
+    <joint name="${prefix}base_link-base_fixed_joint" type="fixed">
         <origin rpy="0 0 0" xyz="0 0 0" />
-        <parent link="wrist_3_link" />
-        <child link="tool0" />
+        <parent link="${prefix}base" />
+        <child link="${prefix}base_link" />
     </joint>
-    <link name="eoat_link">
-        <visual>
-            <geometry>
-                <mesh filename="package://crs_support/meshes/end_effector/visual/ee_v2.stl" />
-            </geometry>
-            <material name="Orange">
-                <color rgba="1.0 0.4 0.0 1.0" />
-            </material>
-        </visual>
-        <collision>
-            <geometry>
-                <mesh filename="package://crs_support/meshes/end_effector/collision/ee_v2_collision_full.stl" />
-            </geometry>
-        </collision>
+    <link name="${prefix}tool0" />
+    <joint name="${prefix}wrist_3_link-tool0_fixed_joint" type="fixed">
+        <origin rpy="0 0 0" xyz="0 0 0" />
+        <parent link="${prefix}wrist_3_link" />
+        <child link="${prefix}tool0" />
+    </joint>
+    <link name="${prefix}eoat_link">
+      <visual>
+          <geometry>
+              <mesh filename="package://crs_support/meshes/end_effector/visual/full_ee_v4.stl" />
+          </geometry>
+          <material name="Orange">
+              <color rgba="1.0 0.4 0.0 1.0" />
+          </material>
+      </visual>
+      <collision>
+          <geometry>
+              <mesh filename="package://crs_support/meshes/end_effector/collision/full_ee_v4_collision.stl" />
+          </geometry>
+      </collision>
+<!--      <inertial>
```

Comment:
Remove if no longer needed

Reply:
This commented section is for our end effector set up with an actual sander. It's set up differently than the WSU setup, so I have it commented in case we want to test actual sanding it is easily accessible.

Reply:
Is this file used by the wsu setup as well?

Reply:
No, this file is specific to the swri setup